### ISSUE TITLE
Crash when attempting to attack

### ISSUE THREAD

**Issue by paradigm369** [CleverRaven/Cataclysm-DDA#27022]
**Describe the bug**  
Upon using the f key to make a range attack or reach attack, the game crashes. This also happens when attempting to throw an item. Essentially game crashes when entering the "targeting" screen

**To Reproduce**  
1. load the attached savegame (error save.zip)
2. Attempt to make a reach attack or range attack
3. Game crashes

**Crash Log**  
CRASH LOG FILE: config/crash.log
VERSION: 0.C-35078-g99ec0d1
TYPE: Signal
MESSAGE: SIGSEGV: Segmentation fault
STACK TRACE:
    @0x5CD515[cataclysm-tiles.exe+0x1CD515]
    @0x5CE0C2[cataclysm-tiles.exe+0x1CE0C2]
    SMPEG_error+0x4B034@0xE4DA20[cataclysm-tiles.exe+0xA4DA20]
    _C_specific_handler+0x98@0x7FFDC48B7C58[msvcrt.dll+0x27C58]
    _chkstk+0x11D@0x7FFDC5AFED3D[ntdll.dll+0x9ED3D]
    RtlWalkFrameChain+0x13F6@0x7FFDC5A66C86[ntdll.dll+0x6C86]
    KiUserExceptionDispatcher+0x2E@0x7FFDC5AFDC6E[ntdll.dll+0x9DC6E]
    SDL_LogCritical+0x12FD6@0xE6E2F6[cataclysm-tiles.exe+0xA6E2F6]
    IMG_LoadWEBP_RW+0x429023@0x1389143[cataclysm-tiles.exe+0xF89143]
    IMG_LoadWEBP_RW+0x19131E@0x10F143E[cataclysm-tiles.exe+0xCF143E]
    initterm_e+0x1AB@0x7FFDC48CA2BB[msvcrt.dll+0x3A2BB]
    @0x5CE05C[cataclysm-tiles.exe+0x1CE05C]
    IMG_LoadWEBP_RW+0x17D848@0x10DD968[cataclysm-tiles.exe+0xCDD968]
    IMG_LoadWEBP_RW+0x48BA23@0x13EBB43[cataclysm-tiles.exe+0xFEBB43]
    IMG_LoadWEBP_RW+0x490B9E@0x13F0CBE[cataclysm-tiles.exe+0xFF0CBE]
    IMG_LoadWEBP_RW+0x4902EC@0x13F040C[cataclysm-tiles.exe+0xFF040C]
    @0x88823E[cataclysm-tiles.exe+0x48823E]
    @0x89CCA7[cataclysm-tiles.exe+0x49CCA7]
    @0xC078CC[cataclysm-tiles.exe+0x8078CC]
    @0x6BC519[cataclysm-tiles.exe+0x2BC519]
    @0x716B5D[cataclysm-tiles.exe+0x316B5D]
    @0x6F9990[cataclysm-tiles.exe+0x2F9990]
    IMG_LoadWEBP_RW+0x4BC319@0x141C439[cataclysm-tiles.exe+0x101C439]
    @0x4013ED[cataclysm-tiles.exe+0x13ED]
    @0x4014FB[cataclysm-tiles.exe+0x14FB]
    BaseThreadInitThunk+0x14@0x7FFDC4643034[KERNEL32.DLL+0x13034]
    RtlUserThreadStart+0x21@0x7FFDC5AD1471[ntdll.dll+0x71471]

**Versions and configuration(please complete the following information):**
 - OS: Windows 10
 - Game Version 0.C-35078-g99ec0d1 build 8266
 - Mods loaded 

[Error save.zip](https://github.com/CleverRaven/Cataclysm-DDA/files/2660323/Error.save.zip)

**Comment by Zireael07** [comment]
Duplicate of #26990, pending fix in #27007 ?

**Comment by AMurkin** [comment]
#27007 is merged already.

**Comment by paradigm369** [comment]
As of Build 8267, the issue still exists. How to download latest experimental?

**Comment by AMurkin** [comment]
Can confirm.
Savefile has `"last_target_pos": [ -2147483648, -2147483648, -2147483648 ]` in it.
How can we workaround a crash after #27007 ?

**Comment by swofle** [comment]
The workaround is to edit the save file.
So change `"last_target_pos": [ -2147483648, -2147483648, -2147483648 ]` to `"last_target_pos": null`
